tincr/cad/util/util.tcl

```diff
@@ -1337,6 +1339,18 @@ proc ::tincr::catch_info { cmd } {
 #  namespace import ::tincr::*
 interp alias {} ::tincr::import_all {} namespace import ::tincr::*
 
+proc ::tincr::print_quiet { message quiet {newline ""} } {
+	if {!$quiet} {
+		puts $newline $message
+	}
+}
+
+proc ::tincr::time_command { command } {
```

Comment:
The report_runtime command already returns the runtime of any given command. Remove time_command and use report_runtime instead.